bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/ApiBridgeHandler.java

```diff
@@ -181,26 +179,22 @@ public Collection<Class<? extends ThingHandlerService>> getServices() {
     }
 
     public synchronized <T> T executeUri(URI uri, HttpMethod method, Class<T> clazz, @Nullable String payload,
-            int retryCount) throws NetatmoException {
+            @Nullable String contentType, int retryCount) throws NetatmoException {
         try {
             logger.trace("executeUri {}  {} ", method.toString(), uri);
 
-            Request request = httpClient.newRequest(uri).method(method).timeout(TIMEOUT_S, TimeUnit.SECONDS)
-                    .header(HttpHeader.CONTENT_TYPE, "application/x-www-form-urlencoded;charset=UTF-8");
```

Comment:
@lolodomo - should "application/x-www-form-urlencoded;charset=UTF-8" be default now if `contentType` is null? I don't see this set anywhere anymore?

Reply:
No it is useless if you have no content. 
Confirmed with my tests (the GET requests are still working).

Reply:
OK, I assumed it was there for a reason for some specific cases, but apparently not. :)

Reply:
But we could discuss the need of an `Accept` header. 
It seems to work well even without it.